gsa/src/web/pages/scanners/dialog.js

```diff
@@ -132,6 +132,8 @@ CertStatus.propTypes = {
   info: PropTypes.object.isRequired,
 };
 
+const SCANNER_DIALOG_MAX_HEIGHT = '300px';
```

Comment:
If this deals with the initial height, please name it SCANNER_DIALOG_INITIAL_HEIGHT. From MAX_HEIGHT I'd expect that the dialog can not be enlarged.